firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandler.cs

```diff
@@ -198,13 +202,84 @@ class WaitForTaskCompletion : CustomYieldInstruction {
         }
       }
     }
+    
+    protected internal FirebaseFirestore TestFirestore() {
+        FirebaseApp app = FirebaseApp.DefaultInstance;
+        return TestCustomFirestore(app, DefaultDatabase);
+    }
+
+    protected internal FirebaseFirestore TestFirestoreWithCustomApp(FirebaseApp app) {
```

Comment:
Can method names all be `TestFirestore` instead of `TestFirestoreWithCustomApp`, `TestFirestoreWithCustomDatabase` and `TestCustomFirestore`.

I think the parameters are self explanatory, and don't need to be described in method name.